addons/web/static/src/js/services/data_manager.js

```diff
@@ -52,7 +51,7 @@ return core.Class.extend({
             }).then(function (action) {
                 self._cache.actions[key] = action.no_cache ? null : self._cache.actions[key];
                 return action;
-            }, this._invalidate.bind(this, this._cache.actions, key));
+            }).catch(() => this._invalidate('actions', key));
```

Comment:
guardedCatch ?

Reply:
These were regular catches, I don't see why we should change it

Reply:
it was a mistake I think